core/scene.go

```diff
@@ -293,12 +292,12 @@ func (sc *Scene) resize(geom math32.Geom2DInt) {
 	sc.NeedsLayout()
 }
 
-// ResizeToContents resizes the scene so it fits the current contents.
+// ResizeToContent resizes the scene so it fits the current content.
 // Only applicable to Desktop systems where windows can be resized.
-// Extra size is added to the amount computed to hold the contents,
+// Optional extra size is added to the amount computed to hold the contents,
 // which is needed in cases with wrapped text elements, which don't
 // always size accurately.
-func (sc *Scene) ResizeToContents(extra image.Point) {
+func (sc *Scene) ResizeToContent(extra ...image.Point) {
```

Comment:
I don't want optional extra; if it is needed, make it irrevocably added for now with a TODO to remove it later once we fix #1307.

Reply:
that is a less future-compatible strategy, once we remove it. if it works without then shouldn't pass anything.